scss/components/_ad-screen.scss

```diff
@@ -1,6 +1,5 @@
 .adScreen {
   cursor: default;
-  z-index: 13000;
```

Comment:
I did a git blame and found the relevant ticket commit, https://jira.corp.ooyala.com/browse/PBW-4017 . The behavior being fixed shouldn't be happening with all the latest code from the various repos.